utils/src/time.rs

```diff
@@ -68,11 +68,9 @@ mod tests {
         let time = SystemTime::UNIX_EPOCH + Duration::from_secs(1) + Duration::from_nanos(999_999);
         assert_eq!(time.epoch_millis(), 1_000);
 
-        // Saturates at u64::MAX
-        let time = SystemTime::UNIX_EPOCH + Duration::from_millis(u64::MAX);
-        assert_eq!(time.epoch_millis(), u64::MAX);
-        let time = time + Duration::from_millis(1);
-        assert_eq!(time.epoch_millis(), u64::MAX);
+        // Add 5 minutes
```

Comment:
Windows’ FILETIME is a 64-bit unsigned count of 100-ns ticks since 1601-01-01, so the largest representable Unix-epoch time is ≈ year +60056:

https://learn.microsoft.com/en-us/windows/win32/sysinfo/file-times